src/libs/Navigation/linkingConfig.js

```diff
@@ -129,6 +129,14 @@ export default {
                     NewWorkspace_Root: ROUTES.WORKSPACE_NEW,
                 },
             },
+
+            // Fullscreen Modal Screens
+            WorkspaceSettings: {
+                screens: {
+                    WorkspaceCard: ROUTES.WORKSPACE_CARD,
+                    exact: true,
```

Comment:
Does this need to be `exact: true` ?

Also curious now about whether a view like this will still be possible as we need to be able to open additional modals above the workspace settings in the "right side pane":

![Screen Shot 2021-06-15 at 2 14 21 PM](https://user-images.githubusercontent.com/32969087/122139188-1d22f580-cde4-11eb-9410-e29db2522ce5.png)

In my initial testing I had no issues with this, but just making sure that you experience the same.

Reply:
Probably this can be tested with the `ROUTES.WORKSPACE_INVITE` to make sure the invite modal can be presented above this view without issue. But I think it should work OK.

Reply:
Yeah, I checked it earlier other modals were opening above it.

Reply:
To clarify, why does it need `exact: true` ? I think it's maybe unnecessary but if that's not correct curious to learn why.

Reply:
Yeah it's not needed. I added it in earlier implementation but now it is not needed.